lib/modules/auth/widgets/header_widget.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+import 'package:podcasty/values/app_theme.dart';
+import 'package:podcasty/values/string_constants.dart';
+
+class HeaderWidget extends StatelessWidget {
+  final bool isLeftOriented;
+
+  const HeaderWidget({
+    Key? key,
+    this.isLeftOriented = true,
+  }) : super(key: key);
+
+  //Header component for Authentication module
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: EdgeInsets.symmetric(
+        vertical: MediaQuery.of(context).size.height * 0.01,
+        horizontal: MediaQuery.of(context).size.width * 0.05,
```

Comment:
Please avoid making repeated calls for inherited widget.

Reply:
Resolved